packages/playwright/src/plugins/gitCommitInfoPlugin.ts

```diff
@@ -35,10 +36,10 @@ export const gitCommitInfo = (options?: GitCommitInfoPluginOptions): TestRunnerP
     name: 'playwright:git-commit-info',
 
     setup: async (config: FullConfig, configDir: string) => {
-      const fromEnv = await linksFromEnv();
-      const fromCLI = await gitStatusFromCLI(options?.directory || configDir, fromEnv);
+      const commitInfo = await linksFromEnv();
+      await enrichStatusFromCLI(options?.directory || configDir, commitInfo);
       config.metadata = config.metadata || {};
-      config.metadata['git.commit.info'] = { ...fromEnv, ...fromCLI };
+      config.metadata['git.commit.info'] = commitInfo;
```

Comment:
Can we use another separator in `git.commit.info` instead of `.`? Otherwise one needs to remember which part should be quoted and can't just type `metadata.git.commit.info.revision`.